app/src/main/java/org/sil/gatherwords/SessionActivity.java

```diff
@@ -80,12 +80,27 @@ protected void onCreate(Bundle savedInstanceState) {
 
         gpsEnabled = false;
 
-        wordListSpinner = findViewById(R.id.word_list_spinner);
-        ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(this,
-                R.array.word_lists, R.layout.world_list_spinner_item);
-        adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-        wordListSpinner.setAdapter(adapter);
-        wordListSpinner.setOnItemSelectedListener(this);
+        /*
+        Display list of files for word spinner from /assets directory.
+         */
+        AssetManager assetManager = getApplicationContext().getAssets();
+
+        try {
+            String[] assets = assetManager.list("wordLists");
+
+            wordListSpinner = findViewById(R.id.word_list_spinner);
+
+            ArrayAdapter<String> adapter = new ArrayAdapter<>(this,
+                    R.layout.world_list_spinner_item, assets);
+
+            adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+            wordListSpinner.setAdapter(adapter);
+            wordListSpinner.setOnItemSelectedListener(this);
+
+        } catch (IOException e) {
+            Log.e("Assets_Msg", "Failed to get assets");
+            e.printStackTrace();
```

Comment:
Do not use print* functions.  You can pass `e` as the last parameter to `Log.e()`.